plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -249,10 +250,14 @@ public void testOptimizeWithColumnMappingMode(String columnMappingMode)
         String physicalName = column.get("metadata").get("delta.columnMapping.physicalName").asText();
         int id = column.get("metadata").get("delta.columnMapping.id").asInt();
 
+
         assertUpdate("INSERT INTO " + tableName + " VALUES 10", 1);
         assertUpdate("INSERT INTO " + tableName + " VALUES 20", 1);
         assertUpdate("INSERT INTO " + tableName + " VALUES NULL", 1);
-        assertUpdate("ALTER TABLE " + tableName + " EXECUTE OPTIMIZE");
+        assertUpdate(Session.builder(getQueryRunner().getDefaultSession())
+                        .setSystemProperty("task_writer_count", "1")
```

Comment:
Instead of setting this session property, can we instead adjust the assertions for the increased file count ?